### ISSUE TITLE
Bogus containers?

### ISSUE THREAD

**Issue by janaxhell** [krateng/maloja#214]
I've just noticed that after deploying Maloja on my docker server, a few unknown and not running containers appeared in Portainer, with references to Maloja:

```
| ecstatic_aryabhata | exited |   | - | krateng/maloja | 2023-06-02 18:14:35 | - | - | administrators
| focused_lamarr | exited |   | - | krateng/maloja | 2023-06-02 17:33:41 | - | - | administrators
| determined_dirac | exited |   | - | krateng/maloja | 2023-06-02 17:31:42 | - | - | administrators
| hungry_vaughan | exited |   | - | krateng/maloja | 2023-06-02 17:28:25 | - | - | administrators
| quizzical_ritchie | exited |   | - | krateng/maloja | 2023-06-02 17:26:52 | - | - | administrators
```
What are these?

**Comment by FoxxMD** [comment]
AFAIK this is entirely dependent on your deployment. The [example compose](https://github.com/krateng/maloja/blob/master/example-compose.yml) file lists only one container. If you are pulling straight from dockerhub its not possible to create more than one container (at a time).

Did you have any trouble initially getting the container to deploy? If the container was recreated without removing the old one those would just be left over containers that can be purged. When you edit a container from portainer you get a popup asking if the old container should be removed, if you did not do that then these would also be left over from that.

You could also just check for yourself...`docker logs quizzical_ritchie` will display output from the container. Or this can be accessed from the container details in portainer.

**Comment by janaxhell** [comment]
Yes, I did have problems, I posted about them here, then solved. But I always updated the stack/compose, not the container. Anyway I was worried someone somehow managed to get into my server and mess it, by the weird look of those names. I have already purged them, so I cannot check anything. I'm still a rookie in selfhosting, but in a year I have tried at least a hundred different applications and something like this never ever happened before. New bit learnt, thanks.

**Comment by FoxxMD** [comment]
The container names are [automatically generated](https://pet2cattle.com/2022/08/docker-container-names-generator) by docker.
